util/src_util/et_dt_validation.c

```diff
@@ -3052,6 +3043,8 @@ UTEST(PLAINTEXT, ENCRYPT_DECRYPT)
     int len = (int)enc_frame_len;
     status = Crypto_TC_ProcessSecurity(ptr_enc_frame, &len, tc_sdls_processed_frame);
     ASSERT_EQ(status, CRYPTO_LIB_SUCCESS);
+
+    Crypto_Shutdown();
```

Comment:
I feel that Crypto_Shutdown() should happen before asserts happen when possible so that the rest of the test suite can function normally in the event an assert fails. I'm not going to hold up the review on this point though.

Reply:
Able to move it a little, but there are multiple calls to apply/process with asserts prior to the next calls.